components/OrgDirectoryTags.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const mockTags = ['Tag'];
+const X = () => (
```

Comment:
I'd consider adjusting this constant name to be something that could be more clear at a glance, such as `xButton`.

Reply:
I renamed it to XButton, because it won't allow me to use the element if it starts with a lowercase character for some reason. Should still be good regardless